app/views/RoomsListView/Header/Header.android.js

```diff
@@ -59,29 +59,41 @@ const Header = React.memo(({
 			</View>
 		);
 	}
-	return (
-		<View style={styles.container}>
-			<TouchableOpacity
-				onPress={onPress}
-				testID='rooms-list-header-server-dropdown-button'
-				disabled={connecting || isFetching}
-			>
-				{connecting ? <Text style={[styles.updating, titleColorStyle]}>{I18n.t('Connecting')}</Text> : null}
-				{isFetching ? <Text style={[styles.updating, titleColorStyle]}>{I18n.t('Updating')}</Text> : null}
-				<View style={styles.button}>
-					<Text style={[styles.server, isFetching && styles.serverSmall, titleColorStyle]} numberOfLines={1}>{serverName}</Text>
-					<Image
-						style={[
-							styles.disclosure,
-							showServerDropdown && styles.upsideDown,
-							{ tintColor: themes[theme].headerTitleColor }
-						]}
-						source={{ uri: 'disclosure_indicator_server' }}
-					/>
-				</View>
-			</TouchableOpacity>
-		</View>
-	);
+        if (showServerDropdown) {
```

Comment:
Hey @BizClimb, it's not the right way to do this.
You can just use `disabled` at `TouchableOpacity` to achieve the same behavior.
And remove the image with the `disclosure`.

Reply:
Ahh, thanks for the clarification and example.